app/src/main/res/layout/status_widget.xml

```diff
@@ -0,0 +1,14 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is layout needed?

Reply:
Well, I've never seen a widget without top-level layout.
According to the [documentation](https://developer.android.com/guide/topics/appwidgets/index.html#layout):
> You must define an initial layout for your widget in XML and save it in the project's `res/layout/` directory. [...] widget layouts are based on RemoteViews, which do not support every kind of layout or view widget.

And there is a reference to this layout in `android:initialLayout` of  `appwidget-provider` (in `res/xml/status_widget_info.xml`).

Reply:
I was talking about the RelativeLayout. It's not needed, right?

Reply:
I'm not sure I get the point.
Do you want to replace it with another layout like `LinearLayout`?
I doubt the widget will work without layout.

Reply:
Just leave the `ImageView`. You don't need to wrap it in a view group.

Reply:
You're right. It works even without layout.